src/app/compliance/reducer.js

```diff
@@ -22,6 +23,7 @@ const defaultState: WhitelistState = {
   criticals: [],
   isTooMany: false,
   listLength: 10,
+  freezeStatus: false,
```

Comment:
it can't be false by default, it should be null by default (which means uninitialized) because you initialize this value via polymath.js